markhor_tracks/src/markhor_hw_interface.cpp

```diff
@@ -52,36 +52,97 @@ void MarkhorHWInterface::setupCTREDrive()
 
   int drive_fl_id, drive_fr_id, drive_rl_id, drive_rr_id = 0;
 
+  SupplyCurrentLimitConfiguration current_limit_config;
+  current_limit_config.enable = true;
+  current_limit_config.currentLimit = 30;
+
   if (nh.getParam("/markhor/markhor_tracks_node/front_left", drive_fl_id) == true)
   {
     front_left_drive = std::make_unique<TalonSRX>(drive_fl_id);
-    front_left_drive->SetNeutralMode(NeutralMode::Brake);
+    front_left_drive->SetNeutralMode(NeutralMode::Coast);
     front_left_drive->SetInverted(true);  // Fix drive orientation
+    front_left_drive->ConfigFactoryDefault();
+    front_left_drive->ConfigSelectedFeedbackSensor(FeedbackDevice::CTRE_MagEncoder_Relative , 0, timeout_ms_);
```

Comment:
Ici je suis curieux, pourquoi est-ce qu'on a choisi de prendre `FeedbackDevice::CTRE_MagEncoder_Relative` au lieu des autres  https://www.ctr-electronics.com/downloads/api/java/html/enumcom_1_1ctre_1_1phoenix_1_1motorcontrol_1_1_feedback_device.html

Reply:
Basically sa c'est resumé a 2 choix : CTRE absolute ou relative, le reste je les ai pas vraiment considéré je penses que c'est pour pkug in du hardware non ctre, j'ai choisi relatif parce que ici le absolute value a vraiment aucune importance etant donne qu'on est en vitesse, sa nous donne quand même une position mais c'est from start pas la position du shaft mecaniquement, c'est pertinent de l'utiliser dans les flippers mais pas dans les tracks, mais en pratique sa change rien

Reply:
Apres verrification c'est un choix entre le output pwm ou quadrature de l'encodeur, quadrature est bcp plus rappide(refresh rate) et peut reach des vitesses beaucoup plus grandes, dans notre cas, je penses que le quadrature est le meilleur choix

Reply:
C'est bon ça me va, est-ce que c'est possible pour toi de me donner les sources sur lesquelles tu te bases pour faire ce choix là svp ?

Reply:
CTRE encoder datasheet vers la fin, ils parlent des  différents modes de l'encodeur

Reply:
![image](https://user-images.githubusercontent.com/12837376/141691298-c3c49de2-959a-4acd-83d6-4f8d47ecfdc7.png)

http://www.ctr-electronics.com/downloads/pdf/Magnetic%20Encoder%20User's%20Guide.pdf

C'est plus clair si on met la référence dans le thread. Pis ouais moi aussi je pense que Quadrature serait préférable.